Answers.md

```diff
@@ -0,0 +1,23 @@
+
+1) Semantic Markup is a way of giving your HTML elements a descriptive meaning that provides your code with improved comprehension to any strangers revising it. In addition, it also improves your websites Google seo rankings. The google search engine is able to return descriptive information of the contents wrapped in a semantically marked element that would otherwise be ommitted in a normal element, such as a <div>.  Some important elements that are considered semantic markup are as follows: 
+
+<nav>
+<footer>
+<header>
+<section>
+<article>
+<address>
+<aside>
+
+
+2) The difference between "display: block" and "display: inline" is the following: 
+
+   * In theory, "display: block" creates an invisible box that covers the entire width of the document. Any info that is on the same line as an element that is classified as "display: block" will be pushed to the following line, below the block. 
+
+   * "display: inline" creates an invisible box that only surrounds the width of the contents inside the classified element. For instance, if the following element: <h1>hello</h1> is classified with "display: inline", the invisible border would only surround "hello", as opposed to the entire width of the document. 
+
```

Comment:
Yup perfect answer, the only thing is a more correct answer for the `display: block` is that it covers the entire width of the container, not just the document.